components/Tabs/Tabs.js

```diff
@@ -2,50 +2,54 @@
 class TabLink {
   constructor(element) {
     // Assign this.element to the passed in DOM element
-    // this.element;
+    this.element = element;
     
     // Get the custom data attribute on the Link
-    // this.data;
-    
+    this.data = this.element.dataset.tab;
+    // console.log(this.data);
     // Using the custom data attribute get the associated Item element
-    // this.itemElement;
-    
+    this.itemElement = document.querySelector(`.tabs-item[data-tab="${this.data}"]`);
+    // console.log(this.itemElement);
     // Using the Item element, create a new instance of the TabItem class
-    // this.tabItem;
+    this.tabItem = new TabItem(this.itemElement);
     
     // Add a click event listener on this instance, calling the select method on click
-
+    this.element.addEventListener('click', () => {this.select()});
```

Comment:
on a single line, you can omit the curly brackets { } for an arrow function.  It's completely optional